[2025-04-11T16:44:34.800+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: query_email_every_2hrs.query_postgres manual__2025-04-11T16:44:33.786064+00:00 [queued]>
[2025-04-11T16:44:34.811+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: query_email_every_2hrs.query_postgres manual__2025-04-11T16:44:33.786064+00:00 [queued]>
[2025-04-11T16:44:34.812+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-11T16:44:34.836+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): query_postgres> on 2025-04-11 16:44:33.786064+00:00
[2025-04-11T16:44:34.848+0000] {standard_task_runner.py:60} INFO - Started process 332 to run task
[2025-04-11T16:44:34.851+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'query_email_every_2hrs', 'query_postgres', 'manual__2025-04-11T16:44:33.786064+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/query_email_dag.py', '--cfg-path', '/tmp/tmp4k0k1352']
[2025-04-11T16:44:34.853+0000] {standard_task_runner.py:88} INFO - Job 57: Subtask query_postgres
[2025-04-11T16:44:34.879+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-11T16:44:34.925+0000] {task_command.py:423} INFO - Running <TaskInstance: query_email_every_2hrs.query_postgres manual__2025-04-11T16:44:33.786064+00:00 [running]> on host 88ab7e40fbb2
[2025-04-11T16:44:35.022+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='query_email_every_2hrs' AIRFLOW_CTX_TASK_ID='query_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-04-11T16:44:33.786064+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-11T16:44:33.786064+00:00'
[2025-04-11T16:44:35.039+0000] {base.py:83} INFO - Using connection ID 'my_postgres_conn' for task execution.
[2025-04-11T16:44:35.101+0000] {sql.py:450} INFO - Running statement: 
                    select * from athletes where noc = 'India' and discipline='Boxing'
                    , parameters: None
[2025-04-11T16:44:35.110+0000] {sql.py:459} INFO - Rows affected: 7
[2025-04-11T16:44:35.135+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-11T16:44:35.151+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=query_email_every_2hrs, task_id=query_postgres, execution_date=20250411T164433, start_date=20250411T164434, end_date=20250411T164435
[2025-04-11T16:44:35.187+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-11T16:44:35.215+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
